src/svc/invocable-scripts/paths/scripts_scriptID.yml

```diff
@@ -4,33 +4,76 @@ get:
     - Data I/O endpoints
     - Invokable Scripts
   summary: Retrieve a script
-  description: Uses script ID to retrieve details of an invokable script.
+  description: |
+    Retrieves an invokable script.
+
+    #### Related Guides
+
+    - [Invoke custom scripts](https://docs.influxdata.com/influxdb/cloud/api-guide/api-invokable-scripts/)
   parameters:
     - in: path
       name: scriptID
       schema:
         type: string
       required: true
-      description: The script ID.
+      description: |
+        The ID of the script to retrieve.
   responses:
     '200':
-      description: The requested script object.
+      description: |
+        Success.
+        The response body contains the script object.
       content:
         application/json:
           schema:
             $ref: '../schemas/Script.yml'
+          examples:
+            $ref: '../schemas/ScriptGetResponse.yml'
+    "401":
+      $ref: "../../../common/responses/AuthorizationError.yml"
+    "500":
+      $ref: "../../../common/responses/InternalServerError.yml"
+    '404':
+      description: |
+        Not found.
+      content:
+        application/json:
+          schema:
+            $ref: "../../../common/schemas/Error.yml"
+          examples:
+            notFound:
+              summary: |
+                The requested script was not found.
+              value: {
+                "code":"not found",
+                "message":"script \"09afa3b220fe400\" not found"
+              }
     default:
-      description: Unexpected error
+      description: Internal server error.
       $ref: '../../../common/responses/ServerError.yml'
 patch:
   operationId: PatchScriptsID
   tags:
     - Invokable Scripts
   summary: Update a script
   description: |
-    Updates properties (`name`, `description`, and `script`) of an invokable script.
+    Updates an invokable script.
+
+    Use this endpoint to modify values for script properties (`description`, and `script`).
+
+    To update a script, pass an object that contains the updated key-value pairs.
+
+    #### Limitations
+
+    - If you send an empty request body, InfluxDB responds with an HTTP `200`
```

Comment:
Does this store an empty script?

Reply:
It does not store an empty script. i'll add an additional statement saying that it does not store an empty script / does not update the script